Sources/PayPalUI/PayPalButton.swift

```diff
@@ -19,18 +19,34 @@ public final class PayPalButton: PaymentButton {
 
     /// Available labels for PayPalButton.
     public enum Label: String {
+        /// Display no label
+        case none
+        
         /// Display "Checkout" on the right side of the button's logo
-        case checkout
+        case checkout = "Checkout"
 
         /// Display "Buy now" on the right side of the button's logo
-        case buyNow
+        case buyNow = "Buy Now"
 
         /// Display "Pay with" on the left side of the button's logo
-        case payWith
+        case payWith = "Pay with"
 
         var label: PaymentButtonLabel? {
             PaymentButtonLabel(rawValue: rawValue)
         }
+        
+        public static func allCases() -> [Label] {
+            [
+                Label.none,
+                Label.checkout,
+                Label.buyNow,
+                Label.payWith
+            ]
+        }
```

Comment:
🧶 : There is a standard library feature that does this, [CaseIterable](https://stackoverflow.com/a/32953212).